package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@keepkey/device-protocol",
-  "version": "6.4.1",
+  "version": "6.5.0",
```

Comment:
This should match the FW release version which will be 7.0.0

Reply:
Fixed.